php/php.editor/src/org/netbeans/modules/php/editor/completion/DocRenderer.java

```diff
@@ -531,11 +532,11 @@ private String composeFunctionDoc(String description, String parameters, String
         }
 
         private String composeParameterLine(PHPDocVarTypeTag param) {
-            return composeParameterLine(param.getTypes(), param.getVariable().getValue(), param.getDocumentation());
+            return composeParameterLine(param.getTypes(), param.getVariable().getValue(), param.getDocumentation(), param);
```

Comment:
Nitpick: we are passing several fields of `param` but then, as the last parameter, we pass `param` itself. Maybe we could pass only 1 argument? I am not sure, just noticed it.

Reply:
Yes, I wanted to do it. The following uses another documentation param...
https://github.com/apache/netbeans/blob/8ad336e64d56b0d27ee8b7a50bfcc47afcded50a/php/php.editor/src/org/netbeans/modules/php/editor/completion/DocRenderer.java#L718-L719

So, I'll change it to the following.
```java
        private String composeParameterLine(PHPDocVarTypeTag param) {
            return composeParameterLine(param, param.getDocumentation());
        }

        private String composeParameterLine(PHPDocVarTypeTag param, String documentation) {
            return composeParameterLine(composeType(param.getTypes(), getTypeKind(param)), param.getVariable().getValue(), documentation);
        }

```
```java
                        String paramDescription = composeParamTagDescription(param, inheritedComments);
                        String paramLine = composeParameterLine(param, paramDescription);
```

Reply:
Fixed.

Reply:
Great, thanks a lot!